File: PairSum.java
import java.io.*; 
import java.util.HashSet; 

class PairSum {
	
	static void findPair(int[] inputArray, int sumToFind) {

		HashSet<Integer> mySet = new HashSet<Integer>();

		for(int index=0; index < inputArray.length; index++){
			int difference = sumToFind - inputArray[index];

			if(mySet.contains(difference)){
				System.out.println(inputArray[index] + ","+ difference);
			}

			mySet.add(inputArray[index]);
		}
	}

	public static void main(String args[]){

		int[] inputArray = {10,15,3,7,25};
		int sum = 13;

		findPair(inputArray, sum);
	}
}

File: pair_sum.py
def find_pair(input_array, sum_to_find):
    my_set = set()

    for num in input_array:
        difference = sum_to_find - num

        if difference in my_set:
            print(f"{num},{difference}")

        my_set.add(num)

if __name__ == "__main__":
    input_array = [10, 15, 3, 7, 25]
    sum_to_find = 13

    find_pair(input_array, sum_to_find)
